CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+4.0.0
+=====
+
++ Drop support for Node.js < 12.
++ Hold back jsen dependency. See https://github.com/ifwe/monocle-api/issues/86
```

Comment:
why go to 4.0.0 now, if we're holding off on the breaking change?

Reply:
Fixed, any applications that had failures due to using older Node.js versions should be pretty obvious, so a major version bump shouldn't be needed for that. The package.json "engines" change added after this also makes this obvious